### PR TITLE
arm64: dts: qcom: motorola-osprey: enable modem

### PR DESCRIPTION
**Pull request by MartijnBraam**

### CODE REVIEW

**Comment by MartijnBraam** [comment]
Fixed a typo so wifi is no longer broken by this. the modem doesn't seem to be starting:

https://paste.sr.ht/~martijnbraam/ae26b3f602dafcbb3038737a1700999c38b0b4ac

File: arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts [hunk 1]
```diff
@@ -22,6 +22,27 @@
...
+			no-map;
+		};
+
+		wcnss_mem: wcnss@8bd00000 {
+			reg = <0x0 0x8bd00000 0x0 0x600000>;
+			no-map;
+		};
+
+		venus_mem: venus@8c300000 {
+			reg = <0x0 0x8c300000 0x0 0x60000>;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts:41]:
```suggestion
			reg = <0x0 0x8c300000 0x0 0x600000>;
```

Comment (MartijnBraam) [reply arch/arm64/boot/dts/qcom/msm8916-motorola-osprey.dts]: venus error is gone now.

**Review by Minecrell — APPROVED** [review]

**Comment by Minecrell** [comment]
Applied, thanks!

**Pull request closed without merging**